argilla/src/argilla/records/_io/_datasets.py

```diff
@@ -19,7 +19,7 @@
 from datasets import IterableDataset, Image
 
 from argilla.records._io._generic import GenericIO
-from argilla.media import pil_to_data_uri
+from argilla._helpers._media import pil_to_data_uri
```

Comment:
I think we should keep this integration because it takes advantage of datasets `map` function and batching to cast images to data uri. The `cast_image` method will then skip it if it gets a data uri.